addon-mirage-support/models/file.js

```diff
@@ -0,0 +1,3 @@
+import { Model } from "ember-cli-mirage";
+
+export default Model.extend({});
```

Comment:
Does ember-cli-mirage support native class syntax?

Reply:
Sadly no. `.extend` is a custom implementation by mirage